vllm/core/scheduler.py

```diff
@@ -914,6 +992,10 @@ def _schedule_default(self) -> SchedulerOutputs:
                                                curr_loras,
                                                enable_chunking=False)
 
+        if len(prefills.seq_groups
+               ) == 0 and self.scheduler_config.policy == "priority":
```

Comment:
nit:
```suggestion
        if not prefills.seq_groups and self.scheduler_config.policy == "priority":
```

Reply:
Keeping the current format for consistency with the previous if check